roles/openshift_health_checker/library/docker_info.py

```diff
@@ -0,0 +1,23 @@
+# pylint: disable=missing-docstring
+"""
+Ansible module for determining information about the docker host.
+"""
```

Comment:
Now I get why wrap this into a module... we need it to run on the inventory hosts, not the control host.
:-)